docs/core/preview3/tools/dotnet-build.md

```diff
@@ -23,33 +23,37 @@ dotnet-build -- Builds a project and all of its dependencies
 
 ## Synopsis
 
-`dotnet build [--help] [--output]  [--framework]  
-    [--configuration]  [--runtime] [--version-suffix]
-    [--build-profile]  [--no-incremental] [--no-dependencies]
-    [<project>]`
+```
+dotnet build <project>
```

Comment:
<project> is optional right? should it be like [<project>]?

also, the options listed below are used with project, correct?